### DISCUSSION TITLE
post/aa610836

### DISCUSSION THREAD

**Discussion by giscus** [StellarX/StellarX.github.io#13] [General]
# post/aa610836

如何成为一个更优秀的程序员？ 要对编程感兴趣，要能从中获得快乐（这是最重要的，如果没有做到的话可以早点转行了😄） 和其他的程序员交流，阅读别人的代码，接手并推进别人的项目。这比看书或者参加培训课程要重要。 多了解开源项目，逛技术网站、论坛 了解至少6种编程语言。包括一种支持抽象类的(例如Java或者C++)，一种支持函数的(Lisp, ML或者Haskell)，一种支持语义抽象的(Lisp)，一

https://stellarx.github.io/post/aa610836.html

**Comment by StellarX** [comment]
hello world!
